src/uwtools/resources/rocoto.jsonschema

```diff
@@ -3,6 +3,82 @@
         "dateTimePattern": {
             "type": "string",
             "pattern": "^(\\d{12}) (\\d{12}) (\\d{2}):(\\d{2}):(\\d{2})$"
+        },
+        "task": {
+            "type": "object",
+            "properties": {
+                "attrs": {
+                    "type": "object",
+                    "properties": {
+                        "name": {
+                            "type": "object"
+                        },
+                        "maxtries": {
+                            "type": "integer",
+                            "minimum": 0
```

Comment:
I'm seeing a contridction between the `schema_with_metatasks` and the documentation [here](https://ufs-srweather-app.readthedocs.io/en/develop/CustomizingTheWorkflow/DefineWorkflow.html#defining-a-task) and [here](http://christopherwharrop.github.io/rocoto/).

It looks like the `schema_with_metatasks` has the `maxtries` listed as:

```
      <!-- Optional task maxtries attribute -->
      <optional>
        <attribute name="maxtries">
          <data type="positiveInteger"/>          
        </attribute>
      </optional>
```
Looking at this it seems like `maxtries` should be an `integer` since it needs to be a positive `integer`, however in the documentation it is listed as a `string`. I'm trying to figure out the best way to approach this.

Reply:
Annoyingly, it may require maximum with minimum to count, maybe something arbitrary like:
```
"exclusiveMinimum": 0
"maximum": 9999
```
I looked at some conditional versions and I don't like how any of them came out

Reply:
There was no upper limit requirement I found reading in the docs, but I used the solution above and it seemed to be exactly what I was looking for. I could also use `maximum: _` for an upper limit if we wanted one. Solution above seemed to be good if you're looking for something.

Reply:
Hey, if it works it works!